samples/Sentry.Samples.AzureFunctions.Worker/local.settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
```

Comment:
Do we need _any_ storage for this sample?

Reply:
Part of the template. Running locally with HTTP and timer trigger, the setting is not needed. I'll remove it.

Reply:
> Do we need any storage for this sample?

Yes. The storage connection string is needed for the timer trigger. The timer uses a storage queue with a visibility timeout. Otherwise, the trigger doesn't work. It shouldn't be a problem, given that both VS2022 and Rider have support for Storage Emulator. Alternatively, there's Azurite that, is often used locally, with the identical development connection string `"UseDevelopmentStorage=true"`.